framework/src/executioners/Transient.C

```diff
@@ -203,6 +217,19 @@ Transient::Transient(const InputParameters & parameters)
     if (_num_steps == 0) // Always do one step in the first half
       _num_steps = 1;
   }
+
+  // Set up relaxation
+  if (_relax_factor != 1.0)
+  {
+    NonlinearSystem & _nl_system = _fe_problem.getNonlinearSystem();
+    System & libmesh_nl_system = _nl_system.system();
+
+    // Store a copy of the previous solution here
+    libmesh_nl_system.add_vector("relax_previous", false);
+
+    // Store a copy of the current solution here
+    libmesh_nl_system.add_vector("relax", false);
```

Comment:
Looks like you do not need this vector, you can directly use the current solution for relaxation.